tests/server/blueprints/cases/test_cases_views.py

```diff
@@ -89,21 +89,58 @@ def test_case(app, user_obj, case_obj, institute_obj):
             # THEN it should return a page
             assert resp.status_code == 200
 
-def test_causatives(app, user_obj, institute_obj):
+def test_causatives(app, user_obj, institute_obj, case_obj):
     # GIVEN an initialized app
     # GIVEN a valid user and institute
+    # There should be no causative variants for test case:
+    assert 'causatives' not in case_obj
+    var1_id = '4c7d5c70d955875504db72ef8e1abe77' # in POT1 gene
```

Comment:
May they never change! 😃

Reply:
> I agree. Are you able to install stuff on stage, I can't :(

I haven't tried yet: a little distracted with the conference, and haven't done it before. I'll see if I can get a hold of @moonso and try it out, otherwise I'd prefer sitting down with you Wednesday/Thursday to have someone look over my shoulder..

Reply:
I'm afraid also you and @moonso don't have the privileges yet. I'll ask at the bioinfo/operations meeeting today!

Reply:
> May they never change! 😃

I accept them as a dogma, they can't change! :)